src/models/room.ts

```diff
@@ -1957,7 +1957,7 @@ export class Room extends ReadReceipt<RoomEmittedEvents, RoomEventHandlerMap> {
             }
         }
 
-        this.on(ThreadEvent.Update, this.onThreadUpdate);
+        this.on(ThreadEvent.NewReply, this.onThreadReply);
```

Comment:
Is this a fix for something?

Reply:
Yes sorry PR took title of the first commit (again), fixed it now, the issue it fixes is in the PR description first line